vault.hcl

```diff
@@ -0,0 +1,196 @@
+consul {
+  # This block specifies the basic authentication information to pass with the
+  # request. For more information on authentication, please see the Consul
+  # documentation.
+  auth {
+    enabled  = true
```

Comment:
We don't have an dependency for consul (which is the section in question). So I don't know if I can remove it, but it shouldn't effect anything.